app/server.go

```diff
@@ -31,13 +31,21 @@ func Server(
 	schoolHandler *http2.SchoolHandler,
 	tagHandler *http3.TagHandler,
 	reviewHandler *http4.ReviewHandler,
-	mw Middleware) *gin.Engine {
+	mwV0 Middleware,
+	mwV1 Middleware,
+	parser ClaimsParser) *gin.Engine {
 	router := gin.Default()
 	router.Use(cors.Default())
-	mapStudentURLs(mw, studentHandler, router)
-	mapSchoolURLs(mw, schoolHandler, router)
+
+	mapStudentURLsV0(mwV0, studentHandler, router)
+	mapSchoolURLsV0(mwV0, schoolHandler, router)
 	mapTagURLs(tagHandler, router)
-	mapReviewURLs(mw, reviewHandler, router)
+	mapReviewURLsV0(mwV0, reviewHandler, router)
+
+	mapStudentURLsV1(mwV1, parser, studentHandler, router)
+	mapSchoolURLsV1(mwV0, parser, schoolHandler, router)
```

Comment:
why do school and review urls V1 have V0 auth?

Reply:
oops, that's a mistake on my part. I'll fix it